modules/microshift-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+//* microshift_support/microshift-etcd.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-configuration_{context}"]
+= Configure `memoryLimitMB` to set parameters for the {product-title} etcd server
+
+By default, etcd will use as much memory as necessary to handle the load on the system. In some memory constrained systems it might be necessary to limit the amount of memory etcd is allowed to use at a given time.
```

Comment:
```suggestion
By default, etcd will use as much memory as necessary to handle the load on the system. In some memory constrained systems, it might be necessary to limit the amount of memory etcd is allowed to use at a given time.
```